gulpfile.js

```
@@ -151,6 +154,13 @@ gulp.task('exportPatterns', ['patternsExport:clean'], () => {
 
 });
 
+gulp.task('distributeSharedConfig', ['css:clean'], (cb) => {
+    distributeSharedConfig.distribute()
+                          .then(cb)
+                          .catch((err) => {
+                          });
```

This silences all errors?